packages/contracts-bedrock/deploy-config/devnetL1-template.json

```diff
@@ -44,7 +44,6 @@
   "eip1559Elasticity": 6,
   "l1GenesisBlockTimestamp": "0x64c811bf",
   "l2GenesisRegolithTimeOffset": "0x0",
-  "l2GenesisDeltaTimeOffset": null,
```

Comment:
`"l2GenesisEcotoneTimeOffset": "0x78",` for testing on devnet. After 2mins, the data source will switch from calldata to blobdata + plasma with celestia backend.